### PR TITLE
Add additional xla parser fuzzer

### PR DESCRIPTION
**Pull request by DavidKorczynski**

### CODE REVIEW

**Comment by DavidKorczynski** [comment]
CC @fcoUnda

**Review by fcoUnda — APPROVED** [review]

File: tensorflow/security/fuzzing/cc/text_literal_reader_fuzz.cc [hunk 1]
```diff
@@ -43,5 +44,13 @@ void FuzzFileRead(std::string data) {
 }
 FUZZ_TEST(TextReaderFuzzer, FuzzFileRead);
 
+void FuzzHLOParseUnverified(std::string data) {
+  auto result = xla::ParseAndReturnUnverifiedModule(data);
+  if (OkStatus() == result.status()) {
```
Comment (fcoUnda) [inline tensorflow/security/fuzzing/cc/text_literal_reader_fuzz.cc:49]: This `if` seems superfluous, did you mean to error out in some of these? (same comment for the other function's ifs)

**Review by fcoUnda — COMMENTED** [review]
You might have wanted to error out in some of these.

File: tensorflow/security/fuzzing/cc/text_literal_reader_fuzz.cc (hunk 1 above)
Comment (DavidKorczynski) [reply tensorflow/security/fuzzing/cc/text_literal_reader_fuzz.cc]: It was an ugly hack -- I substituted for `IgnoreError` now

**Review by fcoUnda — COMMENTED** [review]
I think you need to add "/tensorflow/compiler/xla/service:hlo_parser" to the BUILD target

**Comment by DavidKorczynski** [comment]
> I think you need to add "/tensorflow/compiler/xla/service:hlo_parser" to the BUILD target

It's working for me as is -- are you unable to build? The CI failures look to be unrelated to this change

**Review by fcoUnda — APPROVED** [review]

**Comment by fcoUnda** [comment]
I need to add it internally for it to work, but I can do it.

**Pull request merged**